ambry-frontend/src/main/java/com.github.ambry.frontend/AmbryBlobStorageService.java

```diff
@@ -184,6 +185,11 @@ public void handlePost(RestRequest restRequest, RestResponseChannel restResponse
       checkAvailable();
       long propsBuildStartTime = System.currentTimeMillis();
       BlobProperties blobProperties = RestUtils.buildBlobProperties(restRequest.getArgs());
+      if (blobProperties.getTimeToLiveInSeconds() + TimeUnit.MILLISECONDS.toSeconds(
+          blobProperties.getCreationTimeInMs()) > Integer.MAX_VALUE) {
+        logger.warn("TTL set to very large value in POST request with BlobProperties {}", blobProperties);
```

Comment:
i don't think this needs to be a warning - will pollute our logs. System is not under distress - would recommend debug since the metric is enough.